packages/pwa-kit-react-sdk/src/utils/opentelemetry.js

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {trace, context, SpanStatusCode} from '@opentelemetry/api'
+import {hrTimeToMilliseconds, hrTimeToTimeStamp} from '@opentelemetry/core'
+import logger from './logger-instance'
+
+const DEFAULT_SERVICE_NAME = 'pwa-kit-react-sdk'
+
+export const OTEL_CONFIG = {
```

Comment:
Let's use the `OTEL_CONFIG` in `packages/pwa-kit-react-sdk/src/utils/opentelemetry-server.js` file.

Remove the DEFAULT_SERVICE_NAME constant from opentelemetry-server.js:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/01da556b2e3a692bb5cfe2aafac41d7d8254339d/packages/pwa-kit-react-sdk/src/ssr/server/opentelemetry-server.js#L15

And replace it with `getServiceName`:
```
import {getServiceName, OTEL_CONFIG} from '../../utils/opentelemetry'

const serviceName = options.serviceName || getServiceName()
...
```